docs/topics/vertx-community-and-product-versions.adoc

```diff
@@ -6,10 +6,10 @@
 Some of the {Vertx} components included in this release are not fully tested to run on OpenShift, and are provided as a xref:vertx-tech-preview-components[limited-supportability Technology Preview].
 These components are based on the community version `3.4.2` of {Vertx}.
 
-.Certified {Vertx} Version
+.Supported {Vertx} Version
 
-The productized {Vertx} version `3.4.2.redhat-006` is certified to run on OpenShift and provided as part of the Red Hat subscription.
-The certified {Vertx} runtime artifacts are available from the link:https://maven.repository.redhat.com/ga/[Red Hat Middleware JBoss General Availability Maven Repository^].
+{Vertx} components version `3.4.2.redhat-006` are supported and provided as part of a Red Hat subscription.
+Supported {Vertx} runtime artifacts are available in the link:https://maven.repository.redhat.com/ga/[Red Hat Middleware JBoss General Availability Maven Repository^].
 
 //RN link TBD
-For a complete list of {Vertx} components provided as part of this release, see the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_application_runtimes/1/html-single/red_hat_openshift_application_runtimes_release_notes/#rn-runtime-components-vertx[Release Notes^].
+For a complete list of supported and Technology preview {Vertx} components available with this release of {ProductShortName}, see the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_application_runtimes/1/html-single/red_hat_openshift_application_runtimes_release_notes/#rn-runtime-components-vertx[Release Notes^].
```

Comment:
Either "Technology Preview" or "technology preview"; "Technology preview" looks weird.

Reply:
Thanks, I meant to spell it with initial capitals, that is as _Technology Preview_:) Will update.